src/app/pages/finish/services/finish-task.service.ts

```diff
@@ -13,4 +13,10 @@ export class FinishTaskService {
       this.schedule.addToCompletedTasks(updatedTask)
     return this.schedule.insertTask(updatedTask)
   }
+
+  updateTaskToReportedCompletion(task): Promise<any> {
```

Comment:
There is a similar function in `src/app/pages/home/services/tasks.service.ts`. Is this different from that ?

Reply:
It's the same but fixed in another pull request.

Reply:
ok